package.json

```diff
@@ -42,6 +42,9 @@
     "chai": "^4.1.2",
     "clang-format": "^1.2.4",
     "mocha": "^5.2.0",
+    "rollup": "^0.64.1",
```

Comment:
Why is this at the top-level? I see the comment in #702, but not sure I understand. How is generating browser.js going to work? Why does it need rollup? Why couldn't be directly emitted by tsc?